### DISCUSSION TITLE
"Further Reading" feature editing

### DISCUSSION THREAD

**Discussion by tomasApo** [cotes2020/jekyll-theme-chirpy#1349] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Built from ~`jekyll-theme-chirpy`~ `chirpy-starter`

### Description

![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/75183079/8da4b9ba-bdfe-4a5a-a7c0-e09fe7b77771)

I wanted  to make adjustments to the 'further reading' feature at the bottom of the post pages. Preferably, I would like to remove it completely. If this is too complex, I would like to change the phrasing to 'Previous and Next' instead.

Any ideas? 

### Operations you have already tried

searched for "NEWER" "OLDER" on my repository to see if anything came up, but no luck

**Comment by cotes2020** [comment, marked as answer]
This is easy to accomplish, you just need to edit the YAML part of the `post` layout, change the order or remove some entries.

https://github.com/cotes2020/jekyll-theme-chirpy/blob/2bf87e0de7928f325811e1bb96cfcaefdf6cf66a/_layouts/post.html#L6-L9

**Reply by tomasApo** [reply]
Thx cotes, I don't have the `_layouts` directory, is this due me using the [Chirpy Starter](https://chirpy.cotes.page/posts/getting-started/#option-1-using-the-chirpy-starter)  instead of the Fork? If so do I need to do a fresh fork or can I simple copy the _layouts/ directory from here?

**Reply by cotes2020** [reply]
> ### How did you create the site?
> Built from `jekyll-theme-chirpy`

Then the description you submitted is not true.

Just copy `post.html` to your directory, for details see <https://jekyllrb.com/docs/themes/#overriding-theme-defaults>

**Reply by tomasApo** [reply]
It worked, thank you @cotes2020 

1. Followed the [tutorial](https://jekyllrb.com/docs/themes/#overriding-theme-defaults) to copy the _layouts/directory from my path to my website directory
2. Delete these entries line 7 to 9
 > This is easy to accomplish, you just need to edit the YAML part of the `post` layout, change the order or remove some entries.
> 
> https://github.com/cotes2020/jekyll-theme-chirpy/blob/2bf87e0de7928f325811e1bb96cfcaefdf6cf66a/_layouts/post.html#L6-L9
